src/irobot/ble_connection.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+from bleak import BleakClient
+from common.checksum import add_checksum_to_list
+from irobot.ble_protocol import drive_forward_distance, drive_distance_packet
+import time
+
+RX = "6e400002-b5a3-f393-e0a9-e50e24dcca9e"
+TX = "6e400003-b5a3-f393-e0a9-e50e24dcca9e"
+ROOT_ADDRESS = "271ECDA9-2770-F9B8-6FC9-878F9E2625E4"
```

Comment:
This address is only for Lillie's computer's bluetooth connection.  It was acquired using [bleak's example code](https://github.com/hbldh/bleak#usage)

```
import asyncio
from bleak import BleakScanner

async def main():
    devices = await BleakScanner.discover()
    for d in devices:
        print(d)

asyncio.run(main())
```